### PR TITLE
Breaking: All icons to SVG Symbols

### PR DESCRIPTION
**Pull request by camjc**

### CODE REVIEW

File: src/styles/alexandria/SvgSymbol.scss [hunk 1]
```diff
@@ -1,7 +1,53 @@
...
+  &-clickable {
+    cursor: pointer;
+    pointer-events: auto;
+  }
+
+  // Passing 'circle' into classSuffixes makes the size apply to the circle, with the icon shrunk to fit.
+  &-circle {
+    @mixin circle-size($diameter) {
+      &.svg-symbol-component-#{$diameter} {
+        $inner-diameter: ceil(sqrt(.5) * $diameter);
```
Comment (OndrejCholeva) [inline src/styles/alexandria/SvgSymbol.scss]: is counting the `sqrt(.5)` better than using `0.70710678118`?

Comment (camjc) [inline src/styles/alexandria/SvgSymbol.scss]: Yep. 0.70710678118 is the most magic of any number (meaningless to anyone reading this).

**Comment by vinteo** [comment]
LGTM

**Comment by vinteo** [comment]
Merging...

**Pull request merged**